chatbot-core/database.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from dataclasses import dataclass, field
+from typing import List
+from uuid import uuid4
+
+from qdrant_client import QdrantClient
+from qdrant_client.http.exceptions import UnexpectedResponse
+from qdrant_client.http.models import Distance, VectorParams
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class QdrantDatabase:
+    """Qdrant Vector Database"""
+
+    collection_name: str
+    host: str = "localhost"
+    port: int = 6333
```

Comment:
Let's not use default values. 
They should be explicitly provided where needed. Default values can hide configuration problems, which may only be noticed when the module is actually used.
```suggestion
    host: str
    port: int
```